admin/bulk-inactive-transfer.vbs

```diff
@@ -72,109 +77,145 @@ Do
 	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 Loop until are_we_passworded_out = false					'loops until user passwords back in
 
+Dialog1 = ""
+BeginDialog Dialog1, 0, 0, 126, 50, "Select the excel row to start"
+  EditBox 75, 5, 40, 15, excel_row_to_restart
+  ButtonGroup ButtonPressed
+    OkButton 10, 25, 50, 15
+    CancelButton 65, 25, 50, 15
+  Text 10, 10, 60, 10, "Excel row to start:"
+EndDialog
+
+DO
+    dialog Dialog1
+    If buttonpressed = 0 then stopscript								'loops until all errors are resolved
+    CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
+LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
+
+excel_row = excel_row_to_restart
```

Comment:
Line 95 - Since you've changed the excel_row variables to excel_row_to_restart you don't need this line. 

Two options: Remove this line or change all the other excel_row_to_restart after this line back to excel_row. You don't need both.